src/pages/interviews/serverlesschats.mdx

```diff
@@ -17,3 +17,124 @@ On this episode, Jeremy and Rebecca chat with Swizec Teller about how to approac
 ></iframe>
 
 [Serverless Chats ep #135](https://www.serverlesschats.com/135/)
+
+## Talking points
+
+- If you can JavaScript, you can back end
+- The future belongs to full stack front end engineers
+- Modern backend is a JavaScript function
+- JavaScript is a great language and anyone who can code JavaScript can pick up any other language
+- A lot of what used to make JavaScript really ugly was interacting with the Dom
+- Front end is more like a full fledged app
+- GraphQL is kind of an escape for backend engineers
+- How should a full stack or how should a front end engineer approach serverless?
+- Database and Variable Naming Conventions: Snake Case vs Camel Case
+- Serverless Handbook for Front End Engineers
+- Architecture principles and how to create good serverless developer experience
+- Your average tutorial can make you a mediocre engineer, how to break out of that cycle
+
+## Quotable Quotes
```

Comment:
I think you need empty lines between quotes so they appear as separate paragraphs